pkg/api/common.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/prometheus/prometheus/promql/parser"
 	"github.com/prometheus/prometheus/storage"
 	"github.com/prometheus/prometheus/util/httputil"
+	multi_tenancy "github.com/timescale/promscale/pkg/multi-tenancy"
```

Comment:
Because in encoding/base64, they both are different terms. In multi-tenancy, it's a single term.

What happens if tomorrow, we do a package like, soft-tenancy and hard-tenancy? Each of such words corresponds to a single term.

Reply:
`pkg/tenancy`
`pkg/tenancy/multi`
`pkg/tenancy/soft`
`pkg/tenancy/hard`

Reply:
soft and hard are parts of multi-tenancy. But I got the point. Thank you.